charts/cp-schema-registry/templates/deployment.yaml

```diff
@@ -151,7 +151,17 @@ spec:
       tolerations:
 {{ toYaml .Values.tolerations | indent 8 }}
       {{- end }}
-      {{- if .Values.affinity }}
       affinity:
-{{ toYaml .Values.affinity | indent 8 }}
-      {{- end }}
+        podAntiAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+            - labelSelector:
+                matchLabels:
+                  app: {{ .Release.Name }}
+              topologyKey: kubernetes.io/hostname
+          preferredDuringSchedulingIgnoredDuringExecution:
+            - weight: 100
+              podAffinityTerm:
+                labelSelector:
+                  matchLabels:
+                    app: {{ .Release.Name }}
+                topologyKey: failure-domain.beta.kubernetes.io/zone
```

Comment:
I think that this key is now deprecated. So still working, but we'll need to change all of those soon. You should try the new one (I don't remember it, sorry)

Reply:
The label still exist on nodes but `topology.kubernetes.io/zone` is the recommended now thanks :+1:

Reply:
Yeah, it's the one I had in mind.
BTW: soon, we'll need to change on all our charts